creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -669,21 +656,44 @@ async function _secondarySyncFromPrimary({
   if (forceResyncConfig?.forceResync) mode = 'force_resync'
   if (forceWipe) mode = 'force_wipe'
 
+  const start = Date.now()
+
+  const secondarySyncFromPrimaryLogger = createChildLogger(genericLogger, {
+    wallet,
+    sync: 'secondarySyncFromPrimary',
+    primary: creatorNodeEndpoint
+  })
+
   const { error, result } = await handleSyncFromPrimary({
     serviceRegistry,
     wallet,
     creatorNodeEndpoint,
     blockNumber,
     forceResyncConfig,
     forceWipe,
-    logContext
+    logContext,
+    start,
+    secondarySyncFromPrimaryLogger
   })
   metricEndTimerFn({ result, mode })
   tracing.setSpanAttribute('result', result)
   tracing.setSpanAttribute('mode', mode)
 
   if (error) {
+    secondarySyncFromPrimaryLogger.error(
+      `Sync complete for wallet: ${wallet}. Status: Error, message: ${
+        error.message
+      }. Duration sync: ${
+        Date.now() - start
+      }. From endpoint ${creatorNodeEndpoint}. Prometheus result: ${result}`
```

Comment:
only change here is the addition of the Prometheus result bit

Reply:
great idea